### PR TITLE
Add Romi to drivetrain project

### LINKED ISSUE #114: Add Romi Vendordep to Project Generation
Currently, the Gyro part of the Romi project is waiting on the Romi vendordep. Once that's out, it needs to be added to the code.

### PR DESCRIPTION
**Pull request by Piphi5**
This doesn't deploy the simgui so the romi isn't fully supported but its a part of the project and also included in the integration tests.

Fixes #114

### CODE REVIEW

File: sysid-projects/drive/src/main/cpp/Robot.cpp [hunk 1]
```diff
@@ -14,7 +14,6 @@
 #include <fmt/format.h>
 #include <frc/ADXRS450_Gyro.h>
 #include <frc/AnalogGyro.h>
-// #include <frc/romi/RomiGyro.h>
```
Comment (calcmogul) [inline sysid-projects/drive/src/main/cpp/Robot.cpp:17]: Why is this commented out?

Comment (Piphi5) [reply sysid-projects/drive/src/main/cpp/Robot.cpp:17]: We moved all the gyro setup code into `sysid-library` so it wasn't needed in the drivetrain project anymore.

Comment (calcmogul) [reply sysid-projects/drive/src/main/cpp/Robot.cpp:17]: Should it be removed then?

Comment (Piphi5) [reply sysid-projects/drive/src/main/cpp/Robot.cpp:17]: Isn't the git diff saying that it was removed?

**Review by calcmogul — APPROVED** [review]

**Pull request merged**